llvm/lib/Transforms/Vectorize/VPlanUtils.h

```diff
@@ -25,6 +25,9 @@ bool onlyFirstLaneUsed(const VPValue *Def);
 /// Returns true if only the first part of \p Def is used.
 bool onlyFirstPartUsed(const VPValue *Def);
 
+/// Returns true if \p Def is only used by scalars.
```

Comment:
I am not sure if scalars is accurate here.  A user may be a wide load, only requesting the first lane.


It should say that only scalar values of Def will be used by all users.